harness/tests/determined/common/experimental/test_experiment.py

```diff
@@ -100,59 +99,6 @@ def test_wait_raises_exception_when_experiment_is_paused(
         expref.wait()
 
 
-@responses.activate
```

Comment:
I think these are still good tests. It's unclear to me if sending `limit=None` to the backend results in _no_ limit or undefined limit (and if that's a contract backend will uphold). If undefined, then the backend _could_ still only return a page.

We're still accounting for this behavior in the code, might as well guarantee it in tests.

Thoughts?

Reply:
added this test back to just verify that we get the full response back.